hydra_redis/querying_mechanism.py

```diff
@@ -526,6 +608,9 @@ def help():
           "show <collection_endpoint> properties")
     print("for classes properties:- show class<class_endpoint> properties")
```

Comment:
Same goes for this as well ( line 606)

Reply:
In line 606, the client shows properties of members of a collection(objects) but in line 609, it shows properties of class endpoints.

Reply:
What properties of class endpoints? Can you provide an example?

Reply:
class properties are like: 'Location' for class Location.
But objects properties are like: ['State'", "'name'", "'model'", "'MaxSpeed'", "'Sensor'] for DroneCollection/1 object.